000_introduction/000_introduction.tex

```diff
@@ -1,8 +1,14 @@
 \chapter{Introduction}\label{introduction}
 
-\section{Problem Statement}
-Section on the tower and how we have interpreted the spec.
-Should include a basic spec for features that we wanted to achieve
+Co-operative survival games refer to a subset of political choice games wherein `players' must work together to overcome disaster, else suffer the consequences through both personal and communal damage. Instances of co-operative survival exist in varying media: computer games (\emph{e.g.} ``Don't Starve,'' ``Rust,'' ``Minecraft''), board games (e.g. Ravine) or sociological constructs (e.g. Reducing Viral Transmission ) all of which exhibit a necessity for collective action, with such notions of survival supported by extensive literature \cite{su10030652}. Survival games are often identified within the context of ``Common Pool Resource Management'' (CPR) \cite{ostromCollectiveAction} \cite{smartHome} problems, where to facilitate co-operative survival, a pool of shared resources must be maintained to help mitigate disaster. In the general case of such survival games, following a disaster, if 1 player dies, all players die.
```

Comment:
Are we happy with this reference citation style? I'm personally ok with it, but want to raise the question.

Reply:
As long we all use /cite{}, we can change the style at the very end of the doc :))